Vizzuality/csn-tool
# Splits about page into about and guidance page. updates necessary files

## Overview

Splits About page into Guidance and About page. Adds respective text under the right pages.

### Checklist

- [x] PR has a description that explains it
- [x] PR includes information for people to test it [e.g.: run `npm install`]
- [x] PR is not 2k commits long... please. :pray:

### To test

Run the project and check that there's a Guidance and a About page and they are different in content.
